### PR TITLE
Adding JavaScript into cards 

### PR DESCRIPTION
**Pull request by jellywiz**
- The following interactions have been implemented:
  - When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
  - When the user clicks (or taps) the close (X) button, the popup disappears.

- In order to associate each project with the popup window details, I have refactored the project section:
  - Used a **JavaScript array to store all of the information for all projects**.
  - For each project following pieces of data is stored in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.
  - Updated the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.
  -Implementation of the popup window.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -4,6 +4,221 @@ const navClose = document.getElementById('close-menu');
...
+    name: 'Data Dashboard',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    feature: 'This is a very basic suuvey that can be modified at your convinience',
+    image: './img/SnapshootPortfolio.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveVersionLink: 'https://github.com/jellywiz/Microverse-Profile-First-Project-Mobile',
+    sourceLink: 'https://github.com/jellywiz/Microverse-Profile-First-Project-Mobile',
+  },
+  {
+    id: 5,
```
Comment (franclobo) [inline main.js:66]: - [ ] There are seven "See Project"  buttons, please 🙏🏽 complete the array.

image

File: main.js [hunk 2]
```diff
@@ -4,6 +4,221 @@ const navClose = document.getElementById('close-menu');
...
+const projectInformation = [
+  {
+    id: 0,
+    name: '',
+    description: '',
+    feature: '',
+    image: './img/SnapshootPortfolio.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveVersionLink: '',
+    sourceLink: '',
```
Comment (franclobo) [inline main.js:23]: - [x] Kindly use the same popup window for all cards: 👇🏽 

- There is a title.
- The paragraph is a "Lorem ipsum"
- The border in the orange buttons are in the same color.
- The close button should be inside the popup window
- The popup window should be responsive, mobile, and desktop versions.

| **Figma template** | **Your design** |
|---|---|
| image | image |
| image | image |

File: main.js [hunk 3]
```diff
@@ -4,6 +4,221 @@ const navClose = document.getElementById('close-menu');
...
+  {
+    id: 1,
+    name: 'Data Dashboard',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    feature: 'This is a very basic suuvey that can be modified at your convinience',
+    image: './img/SnapshootPortfolio.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveVersionLink: 'https://github.com/jellywiz/Microverse-Profile-First-Project-Mobile',
+    sourceLink: 'https://github.com/jellywiz/Microverse-Profile-First-Project-Mobile',
+  },
```
Comment (franclobo) [inline main.js:34]: - [ ] Kindly use the same popup window for all cards: 👇🏽 
- [ ] When I scroll down in the popup window I should see the popup's content and the back window should be fixed. I think you could use `overflow: scroll;` in the styles.

- There is a title.
- The paragraph is a "Lorem ipsum"
- The border in the orange buttons are in the same color.
- The close button should be inside the popup window
- The popup window should be responsive, mobile, and desktop versions.

| **Figma template** | **Your design** |
|---|---|
| image | image |
| image | image |

File: main.js [hunk 4]
```diff
@@ -4,6 +4,221 @@ const navClose = document.getElementById('close-menu');
...
+  {
+    id: 2,
+    name: 'Technical Documentation',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    feature: 'A fixed left panel that uses the complete screen hieght and the rest of the screen is for the documentation and detail information',
+    image: './img/SnapshootPortfolio.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveVersionLink: 'https://github.com/jellywiz/Microverse-Profile-First-Project-Mobile',
+    sourceLink: 'https://github.com/jellywiz/Microverse-Profile-First-Project-Mobile',
```
Comment (franclobo) [inline main.js:43]: - [ ] As I said above 👆🏽 please correct the content of the popup window.

File: main.js [hunk 5]
```diff
@@ -4,6 +4,221 @@ const navClose = document.getElementById('close-menu');
...
+  },
+  {
+    id: 3,
+    name: 'Technical Documentation',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    feature: 'A page that uses all the inpt types, and filters for them ina good shape and send results.',
+    image: './img/SnapshootPortfolio.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveVersionLink: 'https://github.com/jellywiz/Microverse-Profile-First-Project-Mobile',
+    sourceLink: 'https://github.com/jellywiz/Microverse-Profile-First-Project-Mobile',
```
Comment (franclobo) [inline main.js:53]: - [ ] As I said above 👆🏽 please correct the content of the popup window.

File: main.js [hunk 6]
```diff
@@ -4,6 +4,221 @@ const navClose = document.getElementById('close-menu');
...
+  },
+  {
+    id: 4,
+    name: 'Data Dashboard',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    feature: 'This is a very basic suuvey that can be modified at your convinience',
+    image: './img/SnapshootPortfolio.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveVersionLink: 'https://github.com/jellywiz/Microverse-Profile-First-Project-Mobile',
+    sourceLink: 'https://github.com/jellywiz/Microverse-Profile-First-Project-Mobile',
```
Comment (franclobo) [inline main.js:63]: - [ ] As I said above 👆🏽 please correct the content of the popup window.

File: main.js [hunk 7]
```diff
@@ -4,6 +4,221 @@ const navClose = document.getElementById('close-menu');
...
+  },
+  {
+    id: 5,
+    name: 'Technical Documentation',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    feature: 'A fixed left panel that uses the complete screen hieght and the rest of the screen is for the documentation and detail information',
+    image: './img/SnapshootPortfolio.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveVersionLink: 'https://github.com/jellywiz/Microverse-Profile-First-Project-Mobile',
+    sourceLink: 'https://github.com/jellywiz/Microverse-Profile-First-Project-Mobile',
```
Comment (franclobo) [inline main.js:73]: - [ ] As I said above 👆🏽 please correct the content of the popup window.

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @jellywiz, 

Good job so far!
You are doing it well 👍🏽 there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- No linter errors ✔️
- Correct Github flow ✔️
- Descriptive PR ✔️
- Professional README file ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.js (hunk 1 above)
Comment (jellywiz) [reply main.js:66]: in the description of the project we are required to finish only the card sections @franclobo please check this https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md

**Review by Davidosky007 — APPROVED** [review]
Hi @jellywiz 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* No linter errors 👍 
* Well written PR description 👍 
* Proper use of Github flow 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**